src/components/Card.js

```diff
@@ -26,15 +51,30 @@ export default class Card {
   }
 
   _handleLikeIcon() {
+    if(this.isLiked()){
+      this._cardElement
+      .querySelector(".card__like-button")
+      .classList.add("card__like-button_active");
+    } else {
     this._cardElement
       .querySelector(".card__like-button")
-      .classList.toggle("card__like-button_active");
+      .classList.remove("card__like-button_active");
+    }
   }
-  _handleDeleteButton() {
+  
+  deleteButton() {
```

Comment:
`deleteCard`, since you removing the card, not the button

Reply:
Done